sdk/storage/Azure.Storage.Blobs/samples/README.md

```diff
@@ -0,0 +1,20 @@
+---
+page_type: sample
+languages:
+- .NET
+products:
+- azure
+- azure-storage
+---
+
+# Azure Storage Blob client library samples for .NET
+
+These sample programs show how to use the .NET client libraries for Azure Storage Blobs in some common scenarios.
+
+|__File Name__|__Description__|
+|-------------|---------------|
+|[Sample01a_HelloWorld](Sample01a_HelloWorld.cs)|upload, download and list blobs|
```

Comment:
Please turn all of the descriptions into sentences with proper casing and punctuation.